platform/platform-configuration-listener/src/main/java/org/codice/ddf/configuration/store/felix/PropertyValueValidator.java

```diff
@@ -0,0 +1,153 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.configuration.store.felix;
+
+import java.util.function.Predicate;
+import java.util.regex.Pattern;
+
+import org.apache.commons.lang.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Validates syntax for the File Handler
+ */
+public class PropertyValueValidator {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(PropertyValueConverter.class);
+
+    private static final Pattern PATTERN = Pattern.compile(",");
+
+    /**
+     * Checks the is the given string is valid syntax for the given configuration value type
```

Comment:
Oops, should just be "Checks if the given string". Apparently I have a thing against the word "if".